screen/features/src/main/java/com/ivy/features/FeaturesViewModel.kt

```diff
@@ -31,14 +31,26 @@ class FeaturesViewModel @Inject constructor(
     }
 
     @Composable
-    fun getFeatures(): ImmutableList<FeatureUi> {
-        val allFeatures = features.allFeatures.map {
-            FeatureUi(
-                name = it.name ?: it.key,
-                description = it.description,
-                enabled = it.asEnabledState()
-            )
+    fun getFeatures(): ImmutableList<Feature> {
```

Comment:
These computations must be `remember` and put inside remember all the keys on which they depend

Reply:
@ILIYANGERMANOV  Not sure around this one, as` it.asEnabledState()` is @Composable, can't use `remember`. Can you provide a demonstration?

Reply:
Got it. Try to just remember the computation where you do the grouping by groups. It's possible but if it's hard feel free to skip this optimization

Reply:
Ok